skopt/space.py

```diff
@@ -10,6 +10,11 @@
 from sklearn.utils.fixes import sp_version
 
 
+class _Ellipsis:
```

Comment:
ah, so there's a reason i did this :)
if i create a list like `[1, 2, 3, '...', 5, 6, 7]` python stringifies it exactly like that, so we get those quote around the ellipsis which is definitely not what we want. The easiest way I could think of to get `[1, 2, 3, ..., 5, 6, 7]` without the quotes is for that ellipsis to be an object whose `__repr__` method returns `...`. Since the only way to create objects in python is to create a class I had to create a class to achieve that.
if you guys can think of a better way to achieve this I'm all ears